process/rating/ratingStepData.go

```diff
@@ -0,0 +1,44 @@
+package rating
+
+import (
```

Comment:
Maybe add one unit test for this structure in order to validate the getters?

Reply:
fixed